vendor/processors/authorizenet.py

```diff
@@ -624,6 +631,66 @@ def subscription_update_price(self, receipt, new_price, user):
         if self.transaction_submitted:
             super().subscription_update_price(receipt, new_price, user)
 
+    def get_customer_id_for_expiring_cards(self, month):
+        paging = apicontractsv1.Paging()
+        paging.limit = 10
+        paging.offset = 1
+        sorting = apicontractsv1.CustomerPaymentProfileSorting()
+        sorting.orderBy = apicontractsv1.CustomerPaymentProfileOrderFieldEnum.id
+        sorting.orderDescending = "false"
+
+        self.transaction = apicontractsv1.getCustomerPaymentProfileListRequest()
+        self.transaction.merchantAuthentication = self.merchant_auth
+
+        self.transaction.searchType = apicontractsv1.CustomerPaymentProfileSearchTypeEnum.cardsExpiringInMonth
+        self.transaction.month = month
+        self.transaction.sorting = sorting
+        self.transaction.paging = paging
+
+        self.controller = getCustomerPaymentProfileListController(self.transaction)
+        self.controller.execute()
+
+        response = self.controller.getresponse()
+
+        self.check_customer_list_response(response)
+        customer_profile_ids = []
+        last_page = 2
+        
+        if self.transaction_submitted:
+            last_page = ceil(response.totalNumInResultSet.pyval / paging.limit) + 1
```

Comment:
if you're rounding up using ceil, i dont think you need to add one... let's discuss!